biome.jsonc

```diff
@@ -0,0 +1,136 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
```

Comment:
This is nice to have, but every time dependatbot updates the biome version I feel someone needs to come and manually update this.

From the docs we can also do:
```json
{
  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json"
}
```

Reply:
yes, not sure if we want too